# DP: Don't add the bsddb multilib path, if already in the standard lib path

Index: b/setup.py
===================================================================
--- a/setup.py	2011-07-26 15:37:22.187571378 +0400
+++ b/setup.py	2011-07-26 15:37:45.319625219 +0400
@@ -977,7 +977,13 @@
             if db_setup_debug:
                 print "bsddb using BerkeleyDB lib:", db_ver, dblib
                 print "bsddb lib dir:", dblib_dir, " inc dir:", db_incdir
-            db_incs = [db_incdir]
+            # only add db_incdir/dblib_dir if not in the standard paths
+            if db_incdir in inc_dirs:
+                db_incs = []
+            else:
+                db_incs = [db_incdir]
+            if dblib_dir[0] in lib_dirs:
+                dblib_dir = []
             dblibs = [dblib]
             # We add the runtime_library_dirs argument because the
             # BerkeleyDB lib we're linking against often isn't in the
